libraries/device-registry/CMakeLists.txt

```diff
@@ -10,4 +13,8 @@ target_link_libraries(gz-sim-yarp-device-registry
     YARP::YARP_os
     YARP::YARP_init)
 
-install(TARGETS gz-sim-yarp-device-registry)
+install(TARGETS gz-sim-yarp-device-registry
+        EXPORT gz-sim-yarp-device-registry)
+
+install(FILES DeviceRegistry.hh 
+        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/gz-sim-yarp-device-registry")
```

Comment:
I updated the install as done in https://github.com/robotology/how-to-export-cpp-library/blob/afb21efb655e7b1cc11636c3d42aad9e02fb626f/src/LibHeaderOnlyTemplateCMake/CMakeLists.txt#L58C1-L63C74 but I'm not sure it is correct.